functions.yml

```diff
@@ -77,8 +81,9 @@ complex_type_manipulation:
       args:
         - name: arr
           doc: The array of numerics
+      returns: A ``spark.ml`` ``SparseVector``
     - name: expand_struct
-      doc: Promote fields of a nested struct to top-level columns. Similar to using struct.* from SQL, but can be used in more contexts.
+      doc: Promotes fields of a nested struct to top-level columns. Similar to using struct.* from SQL, but can be used in more contexts.
```

Comment:
Wrap `struct.*` in backticks to clarify that it's SQL code

Reply:
Done